storage/indexer/src/db_ops.rs

```diff
@@ -17,6 +17,15 @@ pub fn open_db<P: AsRef<Path>>(db_path: P, rocksdb_config: &RocksdbConfig) -> Re
     )?)
 }
 
+pub fn open_tailer_db<P: AsRef<Path>>(db_path: P, rocksdb_config: &RocksdbConfig) -> Result<DB> {
```

Comment:
can we use the same db as the table info?

Reply:
1. use multiple rocksdb to write in parallel
2. decoupled from existing db and won't mess up the existing one if anything go wrong. we can just recover the tailer db.

Reply:
@msmouse what's your opinion here?

Reply:
I'm fine either way. I just don't like calling it "tailer_db" too much 😂

Reply:
could you give some reasons of using the same db?  I listed my reasons above. open existing dbs and adding new data is risky and not bring any clear advantage in my mind.

Reply:
I in general prefer to have less things. However given probably only a few people will turn on the flag and we think it will eventually go away, I don't have a strong opinion here.

I don't like the name either.

Reply:
I should have renamed the db to `const TAILER_DB_NAME: &str = "index_async_db_tailer_db";` what is better name in your mind? I will use it

Reply:
@msmouse suggest a name please.

Reply:
I feel "tailer" is weird in the name. Maybe just something like internal_indexer_db.

Reply:
"internal_indexer_db" works for me